src/style.css

```diff
@@ -1,3 +1,69 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --text-color: #00acdc;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  line-height: 14px;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-inline: 50px;
+}
+
+/*
+.header_logo {
+
+} */
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
```

Comment:
It is better to set these properties to the entire page in the ``body`` or ``html`` tag.
To avoid duplicating code.